backends-velox/src/main/scala/org/apache/spark/sql/execution/VeloxBuildSideRelation.scala

```diff
@@ -33,4 +37,16 @@ case class VeloxBuildSideRelation(output: Seq[Attribute], batches: Array[Array[B
 
   override def asReadOnlyCopy(broadCastContext: BroadCastHashJoinContext
                              ): VeloxBuildSideRelation = this
+
+  override def broadcastMode: BroadcastMode = mode
+
+  /**
+   * Convert broadcasted value to Iterator[InternalRow].
+   *
+   * @return
+   */
+  override def convertColumnarToInternalRow: (Long, Iterator[InternalRow]) = {
+    // convert batches: Array[Array[Byte]] to Iterator[InternalRow] with row num.
+    (0, Iterator.empty)
+  }
```

Comment:
should be able to ref below:
https://github.com/oap-project/gazelle_plugin/blob/main/native-sql-engine/core/src/main/scala/com/intel/oap/execution/DataToArrowColumnarExec.scala#L82-L88